pyproject.toml

```diff
@@ -1,38 +1,78 @@
 [build-system]
 # Defined by PEP 518
 requires = [
-  "setuptools>=42",
-  "wheel",
-  "numpy",
   "Cython",
-  "setuptools_scm[toml]>=6.0"
+  "oldest-supported_numpy",
+  "setuptools>=61",
+  "setuptools_scm[toml]>=7",
+  "wheel",
 ]
 # Defined by PEP 517
 build-backend = "setuptools.build_meta"
 
+[project]
+authors = [
+    {name = "Stratify Contributors", email = "scitools.pub@gmail.com"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "License :: OSI Approved :: BSD License",
```

Comment:
repeat of line 20

Reply:
See [788e044](https://github.com/SciTools/python-stratify/pull/79/commits/788e04469e7b533ca5d74b79a83b030a296bec81)